chart/elastalert2/templates/deployment.yaml

```diff
@@ -48,10 +57,21 @@ spec:
         command:
 {{ toYaml .Values.command | indent 10 }}
 {{- end }}
-{{- if .Values.args }}
+
+{{- if .Values.prometheusPort }}
```

Comment:
This looks like it can be improved to eliminate duplicate code, which will avoid future problems as the chart continues to evolve. I would expect it to look like this:

```
{{- if or .Values.args .Values.prometheusPort }}
        args:
{{- if .Values.args }}
{{ toYaml .Values.args | indent 10 }}
{{- end }}
{{- if .Values.prometheusPort }}
{{- $enableportlist := list "--prometheus_port" (.Values.prometheusPort | toString) }}
{{ toYaml $enableportlist | indent 10 }}
{{- end }}
{{- end }}
```

Reply:
Good question. We can proceed as you have it now and consider adopting a simpler approach if we need to enhance this args area in the future.